packages/grafana-data/src/dataframe/utils.ts

```diff
@@ -12,6 +12,28 @@ export function isTimeSeriesFrame(frame: DataFrame) {
   // and at least one number field
   const timeField = frame.fields.find((field) => field.type === FieldType.time);
   const numberField = frame.fields.find((field) => field.type === FieldType.number);
+
+  // There are certain query types in which we will
+  // get times but they will be the same or not be
+  // in increasing order. To have a time-series the
+  // times need to be ordered from past to present
+  if (timeField !== undefined) {
+    let greatestTime: number | null = null;
+
+    for (let i = 0; i < timeField.values.length; i++) {
```

Comment:
might be worthwhile to limit to checking the first X values

Reply:
I put functionality to add a window and set it to 100